pkg/controller/operators/olm/requirements_test.go

```diff
@@ -453,6 +474,58 @@ func TestRequirementAndPermissionStatus(t *testing.T) {
 			},
 			expectedError: nil,
 		},
+		{
+			description: "RequirementNotMet/NotEstablishedCRDVersion",
+			csv: csv("csv1",
+				namespace,
+				"",
+				installStrategy("csv1-dep", nil, nil),
+				[]*v1beta1.CustomResourceDefinition{crdWithoutStatus("c1", "v2")},
+				nil,
+				v1alpha1.CSVPhasePending,
+			),
+			existingObjs: nil,
+			existingExtObjs: []runtime.Object{
+				crdWithoutStatus("c1", "v2"),
+			},
+			met: false,
+			expectedRequirementStatuses: map[gvkn]v1alpha1.RequirementStatus{
+				{"apiextensions.k8s.io", "v1beta1", "CustomResourceDefinition", "c1group"}: {
+					Group:   "apiextensions.k8s.io",
+					Version: "v1beta1",
+					Kind:    "CustomResourceDefinition",
+					Name:    "c1group",
+					Status:  v1alpha1.RequirementStatusReasonNotAvailable,
+				},
+			},
+			expectedError: nil,
+		},
+		{
+			description: "RequirementNotMet/NamesConflictedCRDVersion",
+			csv: csv("csv1",
+				namespace,
+				"",
+				installStrategy("csv1-dep", nil, nil),
+				[]*v1beta1.CustomResourceDefinition{crdWithoutStatus("c1", "v2")},
```

Comment:
I think I'm missing what makes this CRD be unavailable? What's the difference between this test and the previous?